homework/21_mongobd_install/21_MongoDB_Install.md

```diff
@@ -0,0 +1,179 @@
+## Установка СУБД MongoDB
+
+### Запуск MongoDB в docker контейнере
+Самый простой способ запуска MongoDB в целях обучения и разработки - это docker контейнер.
+Согласно документации на официальном сайте, для запуска MongoDB достаточно последовательно выполнить следующие команды:
+
+```bash
+docker pull mongodb/mongodb-community-server:latest
+
+docker run --name mongodb -p 27017:27017 -d mongodb/mongodb-community-server:latest
+```
+
+После скачивания и запуска контейнера можно проверить его наличие в списке запущенных, статус и проброшенные порты.
+
+```
+>docker container ls
+
+CONTAINER ID   IMAGE                                     COMMAND                  CREATED       STATUS         PORTS                      NAMES  
+0dc9a80ee7eb   mongodb/mongodb-community-server:latest   "python3 /usr/local/…"   2 hours ago   Up 4 seconds   0.0.0.0:27017->27017/tcp   mongodb
+```
+
+Для подключения к запущенной СУБД удобнее всего использовать утилиту mongosh с указанием проброшенного порта. 
+
+```
+mongosh --port 27017
+```
+
+При успешном подключении будет автоматически выбрана база test.
+
+###Заполненеие данными
+Для создания и заполнения новой БД используются следующие команды:
+
+```
+use playground
+
+db.clients.insertMany([ 
+  {"_id": 1, "surname": "Иванов", "first_name": "Иван", "patronymic": "Иванович", "phone": "111111111" },
+  {"_id": 2, "surname": "Силиванов", "first_name": "Пётр", "patronymic": "Петрович", "phone": "222222222" },
+  {"_id": 3, "surname": "Сидоров", "first_name": "Сидор", "patronymic": "Сидорович", "phone": "333333333" }
+] )
+
+db = {

```

Comment:
MongoshInvalidInputError: [COMMON-10002] Cannot reassign 'db' to non-Database type